### PR TITLE
Adding Beaverton to the hometown files

### PR DESCRIPTION
**Pull request by maksschmidt**
This Pull Request was created to add the Beaverton file to the class repo. I'll merge this when I have one :+1:

### CODE REVIEW

File: maksschmidt-beaverton.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Beaverton,OR
+
+### Restaurants
+
+- Decarli
+- Nonna Emilia
```
Comment (CSTaylorJr) [inline maksschmidt-beaverton.md:6]: Is Nonna's _really_ in Beaverton?

**Comment by lichenmike** [comment]
Nonna's official address confirmed:
17210 SW Shaw St, Beaverton, OR 97007

**Comment by swkar** [comment]
good to go! :+1:

**Comment by jdipika87** [comment]
👍

**Pull request merged**